### ISSUE TITLE
Undefined class constant 'INTERPOLATE_BICUBIC'

### ISSUE THREAD

**Issue by wallenium** [nextcloud/server#2580]
### Steps to reproduce
1. No idea, appears in the error log once visiting the page

### Error Message
`Error: Undefined class constant 'INTERPOLATE_BICUBIC'`

Nextcloud 11.0.0.6, PHP 7.0.10, Database MariaDB
Updated NC10.0.1 to RC1 by using the beta channel

**Are you using encryption:** yes

#### Nextcloud log (data/nextcloud.log)
```
{"reqId":"WEl0uiUR4EMAADVUlzIAAAAF","remoteAddr":"194.77.12.2","app":"PHP","message":"Undefined class constant 'INTERPOLATE_BICUBIC' at \/home\/www\/nextcloud\/apps\/theming\/lib\/IconBuilder.php#162","level":3,"time":"2016-12-08T14:56:59+00:00","method":"GET","url":"\/index.php\/apps\/theming\/favicon\/files?v=4","user":"--","version":"11.0.0.6"}

```

**Comment by nickvergessen** [comment]
As per http://php.net/manual/en/imagick.constants.php

> This constant is available if Imagick has been compiled against ImageMagick version 6.3.2 or higher. 

@juliushaertl I guess we should fallback to `FILTER_LANCZOS` which was used before, if the constant does not exist:
https://github.com/nextcloud/server/commit/797e7614925d10f8a11ae239f2122497d0f01fc2

**Comment by juliushaertl** [comment]
fixed with #2584
